omc3/tune_analysis/kick_file_modifiers.py

```diff
@@ -331,10 +326,10 @@ def convert_bbs_kickdataframe(df: tfs.TfsDataFrame) -> tfs.TfsDataFrame:
         if COEFFICIENT.format(order=1) in key:
             df.headers[key] = df.headers[key] * 1e6  # inverse um to inverse m
 
-        # use uncorrected error columns for corrected data
-        if ERR in key:
-            new_key = key.replace(ERR, f"{ERR}{CORRECTED}")
-            df.headers[new_key] = df.headers[key]
+        # # use uncorrected error columns for corrected data
+        # if ERR in key:
+        #     new_key = key.replace(ERR, f"{ERR}{CORRECTED}")
+        #     df.headers[new_key] = df.headers[key]
```

Comment:
why still in but commented out?

Reply:
It probably can be removed, but I was not sure if I cover all versions of the old data.